docs/dev/examples.md

```diff
@@ -0,0 +1,28 @@
+# Examples
+
+This document gives an overview of example structure.
```

Comment:
I would elaborate a bit, with something like: [Examples](link-to-examples-on-github) are tutorials containing more or less elaborated usages of Concrete, to showcase some functionalities and some practical use. They are either given as Python codes or as Jupyter notebooks.

Reply:
I'll update it with:

This document gives an overview of the structure of the examples, which are tutorials containing more or less elaborated usages of Concrete, to showcase its functionality on practical use cases. Examples are either provided as a Python script or a Jupyter notebook.